tool/bash/config_autocomplete.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+__gamer_configure_autocomplete() {
+
+    local options option_array option_array2
```

Comment:
Add comments to explain `option_array` and `option_array2` for improving maintainability if you think that this is needed.